pex/interpreter.py

```diff
@@ -308,41 +287,19 @@ def all(cls, paths=None):
     return cls.filter(cls.find(paths))
 
   @classmethod
-  def _parse_extras(cls, output_lines):
-    def iter_lines():
-      for line in output_lines:
-        try:
-          dist_name, dist_version, location = line.split()
```

Comment:
For historical reference, this line was broken, as `location` is a path, and might have embedded spaces. Should be `line.split(' ', 2)`.